sdk/lib/core/uri.dart

```diff
@@ -568,7 +568,7 @@ abstract interface class Uri {
   /// ```dart import:convert
   /// final uri =
```

Comment:
Consider changing this example to 

```dart
final uri =
     Uri.parse('https://example.com/api/fetch?limit=10&limit=20&max=100');
print(jsonEncode(uri.queryParametersAll)); // {"limit":["10","20"],"max":["100"]}
```

Which would show case the difference of `queryParametersAll`.

Reply:
Right, that's better. I also added `limit=30` to make it more aligned with `queryParameters` example. @mraleph